### DISCUSSION TITLE
How can I notice to connected from other device?

### DISCUSSION THREAD

**Discussion by ssaattww** [nanoframework/Home#1161] [Q&A] [answered]
I am creating HID service.
I want to notice connectted and modify descriptor value and notify.
How can I do? thanks.

**Comment by Ellerbach** [comment, marked as answer]
@ssaattww can you give a bit more context? HID is USB or Bluetooth?
If Bluetooth, see #1144 
Once solve, that should be able to unlock your scenario.

**Reply by ssaattww** [reply]
I use Bluetooth.
I saw #1144. The issue is not solved?

**Reply by Ellerbach** [reply]
It is not solved yet.

**Reply by ssaattww** [reply]
Thank you!
